src/containers/pages/MDAPoint/SchoolReports/index.tsx

```diff
@@ -0,0 +1,209 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+import { Col, Row } from 'reactstrap';
+import { Store } from 'redux';
+import {
+  GenericSupersetDataTable,
+  GenericSupersetDataTableProps,
+} from '../../../../components/GenericSupersetDataTable';
+import HeaderBreadcrumb, {
+  Page,
+} from '../../../../components/page/HeaderBreadcrumb/HeaderBreadcrumb';
+import {
+  SUPERSET_MDA_POINT_REPORTING_JURISDICTIONS_DATA_SLICES,
+  SUPERSET_MDA_POINT_SCHOOL_REPORT_DATA,
+} from '../../../../configs/env';
+import { HOME, MDA_POINT_SCHOOL_REPORT_TITLE } from '../../../../configs/lang';
+import {
+  HOME_URL,
+  MDA_POINT_SCHOOL_REPORT_URL,
+  REPORT_MDA_POINT_PLAN_URL,
+} from '../../../../constants';
+import { RouteParams } from '../../../../helpers/utils';
+import supersetFetch from '../../../../services/superset';
+import { getGenericJurisdictionsArray } from '../../../../store/ducks/generic/jurisdictions';
+import MDAPointSchoolReportReducer, {
+  FetchMDAPointSchoolReportAction,
+  makeMDAPointSchoolReportsArraySelector,
+  reducerName as MDAPointSchoolReportReducerName,
+  SchoolReport,
+} from '../../../../store/ducks/generic/MDASchoolReport';
+
+/** register the MDA point school report definitions reducer */
+reducerRegistry.register(MDAPointSchoolReportReducerName, MDAPointSchoolReportReducer);
+
+const slices = SUPERSET_MDA_POINT_REPORTING_JURISDICTIONS_DATA_SLICES.split(',');
+interface SchoolReportsProps extends GenericSupersetDataTableProps {
+  pageTitle: typeof MDA_POINT_SCHOOL_REPORT_TITLE;
+  pageUrl: string;
+  prevPage: Page | null;
+}
+
+const tableHeaders = [
+  'Age Range',
+  'Total SACs Registered',
+  'MMA coverage',
+  'MMA Coverage (%)',
+  'SACs refused',
+  'SACs sick/pregnant/contraindicated',
+  'ADR reported (%)',
+  'ADR severe (%)',
+  'Alb tablets distributed',
+];
+
+const SchoolReportsList = (props: SchoolReportsProps) => {
+  const {
+    pageTitle,
+    pageUrl,
+    supersetSliceId,
+    fetchItems,
+    service,
+    data,
+    headerItems,
+    tableClass,
+    prevPage,
+  } = props;
+
+  const homePage = {
+    label: HOME,
+    url: HOME_URL,
+  };
+
+  const pages = prevPage ? [homePage, prevPage] : [homePage];
+  const breadcrumbProps = {
+    currentPage: {
+      label: pageTitle,
+      url: pageUrl,
+    },
+    pages,
+  };
```

Comment:
The breadcrumb needs to contain the full path to this report i.e. it should be `Home > MDA Point Reporting > GS MDA-Point 2020-05-26 test > Lusaka > Mtendere > REVEAL TEST` instead of just `Home > Mtendere > REVEAL TEST`

Also note that in the breadcrumb we should have just `REVEAL TEST` instead of `MDA Point School Reporting: REVEAL TEST`.  (let us leave out the "MDA Point School Reporting" in the breadcrumb)